visidata/clipboard.py

```diff
@@ -106,6 +110,14 @@ def save(self, vs, filetype):
                 close_fds=True)
             p.communicate()
 
+def pasteFromClipboard(col_idx, row_idx, sheet):
+    text = pyperclip.paste()
+    for t in text.split('\n'):
+        try:
+            sheet.columns[col_idx].setValue(sheet.rows[row_idx], t)
+        except IndexError:
+            break
```

Comment:
I can see how this `try/except` is useful for when pasting goes past the end of the sheet.  I think it would be better as an `if` though, so that any `IndexError`s caused by setValue aren't swallowed up.  Otherwise I think this should work!

Reply:
sounds good!